packages/math/src/const.ts

```diff
@@ -25,25 +25,25 @@ export const RAD_TO_DEG = 180 / Math.PI;
  */
 export const DEG_TO_RAD = Math.PI / 180;
 
+export const enum SHAPES {
+    POLY = 0,
+    RECT = 1,
+    CIRC = 2,
+    ELIP = 3,
+    RREC = 4,
+}
 /**
  * Constants that identify shapes, mainly to prevent `instanceof` calls.
  *
  * @static
  * @constant
  * @name SHAPES
  * @memberof PIXI
- * @type {object}
+ * @type {enum}
  * @property {number} POLY Polygon
  * @property {number} RECT Rectangle
  * @property {number} CIRC Circle
  * @property {number} ELIP Ellipse
  * @property {number} RREC Rounded Rectangle
+ * @enum {number}
  */
```

Comment:
I think this belongs above the `SHAPES` enum, and doesn't need any of these doc `@` values.

Reply:
The reason for this being below is that the jsdoc gets stripped out when compiled. If you know how to stop this from happening another way i can change it